.github/workflows/CD.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and Release k-NN
+on:
+  push:
+    tags: 
+      - v*
+
+jobs:
+  Build-k-NN:
+    strategy:
+      matrix:
+        java: [12]
+
+    name: Build and Release k-NN Plugin
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout k-NN
+        uses: actions/checkout@v1
+        
+      - name: Configure AWS
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      
+      - name: Setup Java ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      
+      - name: Run build
+        run: |
+          ./gradlew build buildDeb buildRpm --console=plain -Dbuild.snapshot=false
```

Comment:
`build buildDeb buildRpm ` can be `build release`.

Reply:
will update this.